### PR TITLE
Article/react conf

### PR DESCRIPTION
**Pull request by NatalieHarmsworth**

### CODE REVIEW

File: _posts/2020-02-27-react-conf-2020.markdown [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+layout: post
+title:  "ReactConf 2020"
+date:   2020-03-02
+tags: React Sydney
+author: Natalie Laing
+image: /img/blog/reactConf/react.png
+---
+
+## Introduction 
+React Conf AU is the first React conference in the southern hemisphere and it didn't disappoint. The quality of speakers was spot on, meet the faces behind the projects that make React what it is today.
```
Comment (Hamo69) [inline _posts/2020-02-27-react-conf-2020.markdown]: > The quality of speakers was spot on, meet the faces behind the projects that make React what it is today.
 
This sentence doesn't make sense. Perhaps it should be:

The quality of speakers was spot on and we got a chance to meet the faces behind the projects that make React what it is today.

Comment (Hamo69) [inline _posts/2020-02-27-react-conf-2020.markdown]: You may want to give an intro as to what this is.. at the moment seems like a blow by blow account of ReactConf or your personal highlights? Or?

File: _posts/2020-02-27-react-conf-2020.markdown [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+## Day 1
+![day 1](/img/blog/reactConf/conf_day1.jpg)
+
+### Keynote css in js
+Max Stoiber (Gatsby) 
+
+
+The father of styled components took us through CSS in js a history.
```
Comment (Hamo69) [inline _posts/2020-02-27-react-conf-2020.markdown]: > took us through CSS in js a history.

Doesn't make sense?

File: _posts/2020-02-27-react-conf-2020.markdown [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+Diana guided us on GitHub's journey from hardcoded hex color values to using variables and styled components.
+
+On the path to the ~~dark side~~ dark mode, Diana and her team realised that colour is relative and simply inverting the colours would not achieve true dark mode.
+
+``` colour is relative ``` _Diana Mounter ReactConf 2020_ 
+
+## (Proto)typing innovation in BBC News
+James Dooley (BBC news labs)
+
+How a team of artists, developers, and engineers get prototypes ready in 6 weeks for the BBC and how this team is always pushing the boundaries to keep the BBC relevant in today's tech-savvy world.
```
Comment (Hamo69) [inline _posts/2020-02-27-react-conf-2020.markdown]: get? Got?

File: _posts/2020-02-27-react-conf-2020.markdown [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+_David Khourshid ReactConf 2020_
+
+## Powerful REST in a GraphQL world
+Tejas Kumar (G2i Berlin)
+
+Tejas is one of those people who just has amazing stage presence, beiing the last talk of the second day could be quite daunting. People are already tired from lunch and it being the last day of the conference. Tejas picked everyone up and showed us how to use REST in GraphQL by demonstrating how to use giphy end to end with type safety. 
+
+``` Innovate against constraints,chose better over bitter and support open source ``` _Tejas Kumar ReactConf 2020_
+
+### Wrapping up
```
Comment (Hamo69) [inline _posts/2020-02-27-react-conf-2020.markdown]: Any key takeaways?
Anything you will apply to your next or current project?

File: _posts/2020-02-27-react-conf-2020.markdown (hunk 2 above)
Comment (NatalieHarmsworth) [reply _posts/2020-02-27-react-conf-2020.markdown]: Sorry, Harry Potter reference. I will muggle proof it.

File: _posts/2020-02-27-react-conf-2020.markdown (hunk 1 above)
Comment (NatalieHarmsworth) [reply _posts/2020-02-27-react-conf-2020.markdown]: ok, more my highlights. Will re write.

File: _posts/2020-02-27-react-conf-2020.markdown (hunk 3 above)
Comment (NatalieHarmsworth) [reply _posts/2020-02-27-react-conf-2020.markdown]: how they currently get their prototypes out. 
current process. 
happy to change.

**Pull request merged**